controller/internal/enforcer/envoyauthorizer/envoyproxy/sds_server.go

```diff
@@ -0,0 +1,687 @@
+package envoyproxy
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"net"
+	"os"
+	"sync"
+	"time"
+
+	"context"
+
+	"go.aporeto.io/trireme-lib/controller/pkg/secrets"
+	"go.aporeto.io/trireme-lib/policy"
+	"go.aporeto.io/trireme-lib/utils/cache"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	envoy_api_v2_auth "github.com/envoyproxy/go-control-plane/envoy/api/v2/auth"
+	envoy_api_v2_core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	sds "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
+
+	"github.com/golang/protobuf/ptypes"
+	//"github.com/gogo/protobuf/types"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	// SdsSocketpath is the socket path on which the envoy will talk to the remoteEnforcer.
+	//SdsSocketpath = "@aporeto_envoy_sds"
+	SdsSocketpath = "127.0.0.1:2999"
+	//SdsSocketpath = "/var/run/sds/uds_path"
```

Comment:
this line can go ... we are definitely never going to go to a unix socket on a file